docs/runbooks/HCOGoldenImageWithNoSupportedArchitecture.md

```diff
@@ -0,0 +1,207 @@
+    2. Add the DICT to the `spec.dataImportCronTemplates` field in the
+       HyperConverged CR. Add the `dataimportcrontemplate.kubevirt.io/enable`
+       annotation with the value `false` to the DICT. Only the DICT name and
+       the annotation are required, in this case
+
+       For example, to disable the `centos-stream10-image-cron` DICT:
+       ```yaml
+       apiVersion: hco.kubevirt.io/v1beta1
+       kind: HyperConverged
+       metadata:
+         name: kubevirt-hyperconverged
+       spec:
+         dataImportCronTemplates:
+         - metadata:
+             name: centos-stream10-image-cron
+             annotations:
+               dataimportcrontemplate.kubevirt.io/enable: 'false'
+         ```
+2. If you have the self-built desired image, that is supported by the nodes in
+   the cluster, you can modify the pre-defined DICT to use your image, adding
+   the DICT to the `spec.dataImportCronTemplates` field in the HyperConverged
+   CR, and modify its `spec.source.registry` field.
+
+   > Tip: you can find the pre-defined DICTs in HyperConverged CR `status.dataImportCronTemplates`
+   > field, as described [above](#diagnosis). Then you can copy the DICT from
+   > there, and modify it in the HyperConverged CR
+   > `spec.dataImportCronTemplates` field.
+
+   Don't forget to set the `ssp.kubevirt.io/dict.architectures` annotation to
+   include all the architectures supported by your image.
+
+   In this case, you'll need to add all the fields of the DICT.
+
+   For example:
+   ```yaml
+   apiVersion: hco.kubevirt.io/v1beta1
+   kind: HyperConverged
+   metadata:
+     name: kubevirt-hyperconverged
+   spec:
+     dataImportCronTemplates:
+     - metadata:
+       annotations:
+         cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+         ssp.kubevirt.io/dict.architectures: arch1,arch2
+       name: centos-stream10-image-cron
+     spec:
+       garbageCollect: Outdated
+       managedDataSource: centos-stream10
+       schedule: "0 */12 * * *"
+       template:
+         spec:
+           source:
+             registry:
+               url: docker://your-registry/your-image:latest
+           storage:
+             resources:
+               requests:
+                 storage: 10Gi
+   ```
+
+### User-defined DataImportCronTemplates
+
+User-defined DICTs are defined in the HyperConverged CR, in the
+`spec.dataImportCronTemplates` field.
+
+First, check what architectures are supported by the image. You can use the
+following command:
+
+```bash
+$ podman manifest inspect your-registry/your-image:latest
+```
+
+See here for
+the [podman manifest inspect documentation](https://docs.podman.io/en/latest/markdown/podman-manifest-inspect.1.html).
+
+If the image is multi architecture manifest (fat manifest), it will include the
+`manifests` field, which is a list of architectures supported by the image. If
+the image is not a multi architecture manifest, you will need to find out what
+is its architecture.
```

Comment:
If the image is not multi architecture, then the image doesnt include the architecture details?

Reply:
it depend how it was built.